dc/s2n-quic-dc/src/socket/recv/pool.rs

```diff
@@ -253,6 +262,8 @@ struct FreeInner {
     regions: Vec<Region>,
     total: usize,
     open: bool,
+    #[cfg(debug_assertions)]
+    active: std::collections::BTreeSet<usize>,
```

Comment:
nit: seems like a HashSet might be better here? AFAICT we don't rely on iteration order? (Maybe for debug printing)...

but either way this is debug-only so seems OK :)

Reply:
Yeah we don't rely on iteration order. I think it's a good change - I can switch to `HashSet` in a future PR.